agenthub/coact_agent/meta/planner_prompt.py

```diff
@@ -0,0 +1,33 @@
+PLANNER_CONSTRUCT_PLAN = """Your role is to construct a multi-stage global plan, providing separate subtask descriptions and expected states for each phase. Ensure that your plan is comprehensive and covers all aspects of the task. Use the following format:
+
+**Subtask 1:**
+- **Subtask**: Describe the first subtask you are about to plan.
+- **Expected State**: Define the expected state of the task after completing the first subtask.
+
+**Subtask 2:**
+- **Subtask**: Describe the second subtask you are about to plan.
+- **Expected State**: Define the expected state of the task after completing the second subtask.
+
+...
+
+**Subtask n:**
+- **Subtask**: Describe the nth subtask you are about to plan.
+- **Expected State**: Define the expected state of the task after completing the nth subtask.
+"""
+
+PLANNER_DECIDE = """The local agent encountered issues with the global planner's global plan and he believes it's necessary to replan globally. Here is the reasons he proposed: {reasons}. Your current task is to decide whether to agree to the re-planning based on the description in the request for a new global plan submitted by the local agent. If you agree, the next action is to ```revise```; if not, the next action is to ```overrule```. Consider the implications of your decision and provide clear reasoning for it. Make sure to give clear and constructive guidance for plan adjustments.
```

Comment:
Wow, this is a mouthful :D 
`The local agent encountered issues with the global planner's global plan and he believes it's necessary to replan globally.`